docs/java-api.md

```diff
@@ -147,6 +147,69 @@ t.newAppend().appendFile(data).commit();
 t.commitTransaction();
 ```
 
+### WriteData
+
+The java API can write data into iceberg table.
+
+First write data to the data file, then submit the data file, the data you write will take effect in the table.
+
+For example, add 1000 rows of data to the table.
```

Comment:
I think we can just say "The following example will demonstrate how to write data files into Iceberg tables". The other details can just be inline comments imo.